debian/control

```diff
@@ -24,6 +24,7 @@ Build-Depends:
  libglib2.0-dev (>= 2.54.0),
  libgsystem-dev,
  libjson-glib-dev (>= 1.2.6-1endless2),
+ libmogwai-schedule-client-0,
```

Comment:
So I think this should be `-dev` since the `.pc` file is in `libmogwai-schedule-client-0-dev`.

Reply:
Argh, of course.

Reply:
Fixup pushed.